Cargo.toml

```diff
@@ -74,3 +75,7 @@ semver = "1.0"
 serial_test = "2.0"
 thiserror = "1.0"
 url = "2.4"
+
+
+[patch.crates-io]
```

Comment:
TODO: remove this after `Address::from_public_key_bytes`

Reply:
Released 0.5.3